imports/client/components/PuzzleListPage.tsx

```diff
@@ -424,37 +428,52 @@ interface PuzzleListPageProps extends PuzzleListPageWithRouterParams {
   canUpdate: boolean;
   allPuzzles: PuzzleType[];
   allTags: TagType[];
+  hunt?: HuntType;
 }
 
 class PuzzleListPage extends React.Component<PuzzleListPageProps> {
   render() {
+    const leadLinks = this.props.hunt?.homepageUrl && (
```

Comment:
once you've asserted hunt is always truthy you can remove the `?` after `hunt` here